### PR TITLE
DDF-1195 Updated Managing.adoc with 'Configuring DDF with New Certificate' section

### PR DESCRIPTION
**Pull request by clockard**
Updated document with content from the 'How-to articals -> Configure DDF with New Certificates' section. Added content form other sources along with some notes/comments.

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf/34)

### CODE REVIEW

**Comment by clockard** [comment]
@stustison
@figliold

File: docs/src/main/resources/Managing.adoc [hunk 1]
```diff
@@ -1168,6 +1168,280 @@ TLS_RSA_WITH_AES_128_CBC_SHA
...
+
+. Start DDF 
++
+[source,bash]
+----
+$> cd <DDF_HOME>/bin
+$> ./ddf
+----
++
+. Go to https://localhost:8993/admin and step through the installer leaving all the defaults.
```
Comment (stustison) [inline docs/src/main/resources/Managing.adoc]: They should update the hostname and port with the actual hostname and desired port for the system they are deploying.

File: docs/src/main/resources/Managing.adoc [hunk 2]
```diff
@@ -1168,6 +1168,280 @@ TLS_RSA_WITH_AES_128_CBC_SHA
...
+org.apache.ws.security.crypto.merlin.keystore.type=jks
+org.apache.ws.security.crypto.merlin.keystore.password=<NewPassword>
+org.apache.ws.security.crypto.merlin.keystore.alias=<Alias>
+org.apache.ws.security.crypto.merlin.keystore.file=etc/keystores/serverKeystore.jks
+----
+
+==== Creating New Certificates
+===== Create private key and certificate signing request
+
+.Create private key
```
Comment (stustison) [inline docs/src/main/resources/Managing.adoc]: We have a script already for doing all of this https://github.com/codice/ddf/blob/master/distribution/ddf-common/src/main/resources/etc/certs/CertNew.sh does that script not work? Also, they should only be doing this step if they don't already have key pair that they want to install. I think the normal use case for an administrator is that they'll have a key pair that they want the system to use.

File: docs/src/main/resources/Managing.adoc (hunk 1 above)
Comment (clockard) [inline docs/src/main/resources/Managing.adoc]: Changed this line as recommended.

File: docs/src/main/resources/Managing.adoc (hunk 2 above)
Comment (clockard) [inline docs/src/main/resources/Managing.adoc]: Broke the creating new certificate section into a creation and importing section. Added notes and links on where to skip to based on what you are trying to do.

**Comment by figliold** [comment]
Review status: 0 of 1 files reviewed, 5 unresolved discussions, some commit checks failed.

---

<sup>**[docs/src/main/resources/Managing.adoc, line 1266 \[r2\]](https://reviewable.io:443/reviews/codice/ddf/34#-JpiZpLOxOwA5QC5Nd7_)** ([raw file](https://github.com/codice/ddf/blob/d1fdba94ffa46ab27795ad7c986a474c8fd3903d/docs/src/main/resources/Managing.adoc#L1266)):</sup>
Should filed be file?

---

<sup>**[docs/src/main/resources/Managing.adoc, line 1270 \[r2\]](https://reviewable.io:443/reviews/codice/ddf/34#-JpiZwXccY_81An0uuED)** ([raw file](https://github.com/codice/ddf/blob/d1fdba94ffa46ab27795ad7c986a474c8fd3903d/docs/src/main/resources/Managing.adoc#L1270)):</sup>
Should filed be file?

---

<sup>**[docs/src/main/resources/Managing.adoc, line 1290 \[r2\]](https://reviewable.io:443/reviews/codice/ddf/34#-Jpi_waXlRjcVW3M7zjn)** ([raw file](https://github.com/codice/ddf/blob/d1fdba94ffa46ab27795ad7c986a474c8fd3903d/docs/src/main/resources/Managing.adoc#L1290)):</sup>
It looks like the above line has change from localhost=localhost,admin to the following in the current ddf distribution: localhost=localhost,group,admin,manager,viewer,webconsole

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/34)

**Comment by clockard** [comment]
Review status: 0 of 1 files reviewed, 5 unresolved discussions, some commit checks pending.

---

<sup>**[docs/src/main/resources/Managing.adoc, line 1266 \[r2\]](https://reviewable.io:443/reviews/codice/ddf/34#-JpiZpLOxOwA5QC5Nd7_)** ([raw file](https://github.com/codice/ddf/blob/d1fdba94ffa46ab27795ad7c986a474c8fd3903d/docs/src/main/resources/Managing.adoc#L1266)):</sup>
It should have been field. This has been fixed in the latest update.

---

<sup>**[docs/src/main/resources/Managing.adoc, line 1270 \[r2\]](https://reviewable.io:443/reviews/codice/ddf/34#-JpiZwXccY_81An0uuED)** ([raw file](https://github.com/codice/ddf/blob/d1fdba94ffa46ab27795ad7c986a474c8fd3903d/docs/src/main/resources/Managing.adoc#L1270)):</sup>
It should have been field. This has been fixed in the latest update.

---

<sup>**[docs/src/main/resources/Managing.adoc, line 1290 \[r2\]](https://reviewable.io:443/reviews/codice/ddf/34#-Jpi_waXlRjcVW3M7zjn)** ([raw file](https://github.com/codice/ddf/blob/d1fdba94ffa46ab27795ad7c986a474c8fd3903d/docs/src/main/resources/Managing.adoc#L1290)):</sup>
Updated this to match the latest config file.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/codice/ddf/34)

**Pull request merged**